cinema_service/urls.py

```diff
@@ -3,9 +3,15 @@
 from django.contrib import admin
 from django.urls import path, include
 
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
+
+
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The inclusion of `debug_toolbar.urls` should be conditional based on the `DEBUG` setting to ensure it's only available in development environments. This can help prevent exposing debug information in production.